assets/css/style.css

```diff
@@ -0,0 +1,9 @@
+.size {
+    width: 100px;
+    height: 100px;
+}
+
```

Comment:
for scaling down images you need to pick either `width` or `height` but not both. if the image ration is anything different than 1:1, you're causing a distort